.travis.yml

```diff
@@ -5,8 +5,6 @@ python:
   - 3.3
   - 3.4
 env:
-  matrix:
-    - DJANGO=django==1.7.3
```

Comment:
But this repository's requirements.txt say it can run in django>=1.7.
So I think we should test django>=1.7. If fail test in django>=1.7, we should fix it, or fix the version of django in requirements.txt.